db/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/RepositoryTest.kt

```diff
@@ -133,18 +135,38 @@ class RepositoryTest : FunSpecWithDb(listOf(InntektsmeldingEntitet, Forespoersel
         }.shouldBeEmpty()
 
         val UUID = randomUuid()
-        val DOK_1 = INNTEKTSMELDING_DOKUMENT.copy(tidspunkt = ZonedDateTime.now().toOffsetDateTime())
+        val DOK_1 = INNTEKTSMELDING_DOKUMENT.copy(tidspunkt = OffsetDateTime.now())
         val JOURNALPOST_1 = "jp-1"
 
         foresporselRepo.lagreForespoersel(UUID.toString(), ORGNR)
         inntektsmeldingRepo.lagreInntektsmelding(UUID.toString(), DOK_1)
-        inntektsmeldingRepo.oppdaterJournalpostId(JOURNALPOST_1, UUID)
+        inntektsmeldingRepo.oppdaterJournalpostId(UUID, JOURNALPOST_1)
         val record = testRepo.hentRecordFraInntektsmelding(UUID.toString())
         record.shouldNotBeNull()
         val journalPostId = record.getOrNull(InntektsmeldingEntitet.journalpostId)
         journalPostId.shouldNotBeNull()
     }
 
+    test("skal _ikke_ oppdatere journalpostId for ekstern inntektmelding") {
```

Comment:
Kunne kanskje lagt til en test (eller bare endret denne) til at vi har både ekstern inntektsmelding OG en innsendt fra simba, og sjekke at kun den simba-innsendte blir oppdatert

Reply:
Done did it.

Reply:
Lol, glemte den viktigste delen 🤦

Reply:
Fikset nå.